website/docs/r/function_action.html.markdown

```diff
@@ -1,154 +1,162 @@
-* `limits` - (Optional, list) A nested block to describe assigned limits. Nested `limits` blocks have the following structure:
-    * `timeout` - The timeout limit to terminate the action, specified in milliseconds. Default value: `60000`.
-    * `memory` - The maximum memory for the action, specified in MBs. Default value: `256`.
-    * `log_size` - The maximum log size for the action, specified in MBs. Default value: `10`.
-* `exec` - (Required, list) A nested block to describe executable binaries. Nested `exec` blocks have the following structure:
-    * `image` - (Optional, string) When using the `blackbox` executable, the name of the container image name.  
-     **NOTE**: Conflicts with `exec.components`.
-    * `init` - (Optional, string) When using `nodejs`, the optional zipfile reference.  
-     **NOTE**: Conflicts with `exec.components`, `exec.image`.
-    * `code` - (Optional, string) The code to execute.  
-    **NOTE**: Conflicts with `exec.components`, `exec.code_path`.
-    * `code_path` - (Optional, string) The file path of code to execute and it supports only .zip extension to create the action.
-    **NOTE**: Conflicts with `exec.components`,`exec.code`.
-    * `kind` - (Required, string) The type of action. You can find supported kinds in the [IBM Cloud Functions docs](https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-runtimes).
-    * `main` - (Optional, string) The name of the action entry point (function or fully-qualified method name, when applicable).  
-    **NOTE**: Conflicts with `exec.components`, `exec.image`.
-    * `components` - (Optional, string) The list of fully qualified actions.  
-    **NOTE**: Conflicts with `exec.code`, `exec.image`,`exec.code_path`.
-* `publish` - (Optional, boolean) Action visibility.
-* `user_defined_annotations` - (Optional, string) Annotations defined in key value format.
-* `user_defined_parameters` - (Optional, string) Parameters defined in key value format. Parameter bindings included in the context passed to the action. Cloud Function backend/API.
-
-## Attributes Reference
-
-In addition to all arguments above, the following attributes are exported:
-
-* `id` - The unique identifier of the action.The id is combination of namespace and actionID delimited by `:` .
-* `namespace` - The name of the function namespace.
-* `version` - Semantic version of the item.
-* `annotations` - All annotations to describe the action, including those set by you or by IBM Cloud Functions.
-* `parameters` - All parameters passed to the action when the action is invoked, including those set by you or by IBM Cloud Functions.
-* `action_id` - Action ID	
-* `target_endpoint_url` - Target endpoint URL of the action.
 
-## Import
+## Argument reference
+Review the argument reference that you can specify for your resource. 
+ 
+- `exec` - (Required, List) A nested block to describe executable binaries.
+
+  Namespace scheme for `exec`:
```

Comment:
Namespace scheme shd be Nested schema...simailrly down also

Reply:
updated the correction.